python/sanlock.c

```diff
@@ -125,7 +146,7 @@ __parse_resource(PyObject *obj, struct sanlk_resource **res_ret)
         if (offset == NULL) {
             res->disks[i].offset = 0;
         } else {
-            res->disks[i].offset = PyInt_AsLong(offset);
+            res->disks[i].offset = PyLong_AsLong(offset);
```

Comment:
Replacing integer with long in places when we can is a great change, please submit it separately.